src/server/index.ts

```diff
@@ -9,28 +9,56 @@ import bodyParser from 'body-parser';
 import { configureStore } from '../shared/store';
 import serverRender from './render';
 import paths from '../../config/paths';
+import { ErrorRequestHandler, Handler, RequestHandler, Request } from 'express-serve-static-core';
+import { Store } from 'redux';
 
 require('dotenv').config();
 
 const app = express();
 
+interface RequestWithStore extends Request {
+    store?: Store;
+}
+
+const errorHandler: ErrorRequestHandler = (err, _, res) =>
```

Comment:
the 4th unused error parameter is necessary here for express to know it's an error handling middleware:
https://expressjs.com/en/guide/error-handling.html#writing-error-handlers

Reply:
No it is not. I explicitely tell it to be an `ErrorRequestHandler`.

Reply:
Yes. And `ErrorRequestHandler` also expects four parameters 😉
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L38

Reply:
Not using a parameter is not forbidden.